core/src/main/groovy/noe/common/utils/InstallerUtils.groovy

```diff
@@ -95,9 +100,25 @@ class InstallerUtils {
       log.error("Zip file $zipSource, exist = ${zipSource.exists()}")
       throw new FileNotFoundException("Zip installation failed, $zipDest doesn't exits")
     }
+    computeSha256Hash(zipDest)
```

Comment:
Rename the function. The purpose of the function is to print the hash sum into a log.